### DISCUSSION TITLE
We look forward to your support for A Reasonably Extra Secure Operating System.

### DISCUSSION THREAD

**Discussion by universish** [nix-community/infra#1494] [Ideas]
I have created a project to build an operating system that is reasonably secure, extra secure and privacy oriented. I have created the repository. At the center of this project is NixOS. The outer armor is Qubes OS. I need a NixOS virtual machine to run inside Qubes OS. At the same time, we need to create NixOS artifacts in other virtual machines that can communicate and transfer data via API with NixOS and other GNU/linux operating systems. We are looking for your support for this. Please contribute to this project to build an operating system that will protect the news sources of journalists, dissidents and journalists living in repressive regimes. 

Our community name: We are from Universe

Project name: 4D Qubes Nexus OS 
Colloquial name: TetraQubes OS. To make it more memorable.
For more information, please visit :
https://github.com/We-are-from-Universe/4D-Qubes-Nexus-OS

**Comment by zimbatm** [comment]
Hi,

To get help, the best place is to ask at <https://discourse.nixos.org/> or on the [NixOS Discord](https://discord.com/invite/2HeKRM3).

Good luck with your idea!
